common/src/main/java/org/dash/wallet/common/ui/CurrencyTextView.java

```diff
@@ -45,25 +41,14 @@ public class CurrencyTextView extends AppCompatTextView {
     private MonetaryFormat format = null;
     private boolean alwaysSigned = false;
     private RelativeSizeSpan prefixRelativeSizeSpan = null;
-    private ScaleXSpan prefixScaleXSpan = null;
-    private ForegroundColorSpan prefixColorSpan = null;
     private RelativeSizeSpan insignificantRelativeSizeSpan = null;
     private boolean applyMarkup = true;
-    private int prefixColor = 0;
```

Comment:
We never use this.
Given that Rostislav wants to remove the variable font size in the currency texts, we might not need this class at all in the future. But for now, I just cleaned up the prefix code.